### ISSUE TITLE
xClusterQuorum: Test-TargetResource functionality with error message: Unknown quorum resource: File Share Witness

### ISSUE THREAD

**Issue by constantinhager** [PowerShell/xFailOverCluster#170]
**Details of the scenario you tried and the problem that is occurring:**
I have installed a Cluster. The Cluster Quorum is installed correctly (I see It in the Fail Over Cluster Manager). If I run the DSC Configuration again I get the following Error:

`PowerShell DSC resource MSFT_xClusterQuorum  failed to execute Test-TargetResource functionality with error message: Unknown quorum resource: File Share Witness
    + CategoryInfo          : InvalidOperation: (:) [], CimException
    + FullyQualifiedErrorId : ProviderOperationExecutionFailure
    + PSComputerName        : SQL1`

`The SendConfigurationApply function did not succeed.
    + CategoryInfo          : NotSpecified: (root/Microsoft/...gurationManager:String) [], CimException
    + FullyQualifiedErrorId : MI RESULT 1
    + PSComputerName        : SQL1`

DSC Config in Attachment
[DSCConfig.txt](https://github.com/PowerShell/xFailOverCluster/files/1628835/DSCConfig.txt)

**Version of the Operating System and PowerShell the DSC Target Node is running:**
Windows Server 2016 Insider Preview
Build Number: 16299
Version: 10.0.16299

PowerShell Version: 5.1.16299.98

**Version of the DSC module you're using, or 'dev' if you're using current dev branch:**
dev

**Comment by johlju** [comment]
I think you hit an error that is solved in the Dev version, but not yet released. See the change log under the Unreleased section; https://github.com/PowerShell/xFailOverCluster/blob/dev/CHANGELOG.md.

Could you verify that is indeed your problem?

**Comment by constantinhager** [comment]
Yes that is indeed my problem. I downloaded the xFailOverCluster Module yesterday. I will test It again when the release is there.

**Comment by shurick81** [comment]
I'm having a similar issue in 1.8.0.0:

```
VERBOSE: [FKV03SPDB]: LCM:  [ End    Test     ]  [[xClusterQuorum]Quorum]  in 0.3130 seconds.
PowerShell DSC resource MSFT_xClusterQuorum  failed to execute Test-TargetResource functionality with error message:
Unknown quorum resource: File Share Witness
    + CategoryInfo          : InvalidOperation: (:) [], CimException
    + FullyQualifiedErrorId : ProviderOperationExecutionFailure
    + PSComputerName        : fkv03spdb

VERBOSE: [FKV03SPDB]: LCM:  [ End    Set      ]
The SendConfigurationApply function did not succeed.
    + CategoryInfo          : NotSpecified: (root/Microsoft/...gurationManager:String) [], CimException
    + FullyQualifiedErrorId : MI RESULT 1
    + PSComputerName        : fkv03spdb
```

The configuration:
```
        xClusterQuorum Quorum
        {
            IsSingleInstance        = 'Yes'
            Type                    = 'NodeAndFileShareMajority'
            Resource                = $mainParameters.SQLWSFCWitnessShare
        }
```

compiled:
```
{
ResourceID = "[xClusterQuorum]Quorum";
 Type = "NodeAndFileShareMajority";
 SourceInfo = "C:\\Projects\\Unionen_Intranet_SharePoint2016Upgrade\\SPUpgrade\\DSC\\WSFC.ps1::70::9::xClusterQuorum";
 IsSingleInstance = "Yes";
 ModuleName = "xFailOverCluster";
 Resource = "\\\\fkv01spadmin\\sqlwscfwitnessprod";
 ModuleVersion = "1.8.0.0";

 ConfigurationName = "WSFC";

};
```

**Comment by johlju** [comment]
Closing this as I think this was resolved in 1.9.0.0. If not, please comment or reopen this issue.
